src/components/InfoPanel.component.js

```diff
@@ -21,6 +23,16 @@ class InfoPanel extends Component {
     setSelectedPOI(activeEvents[curIndex + 1].id)
   }
 
+  onClickEdit() {
+    const { activeEvents, selectedEvent, setSelectedPOI } = this.props
```

Comment:
Don't really need this line. Even though it's a todo, having unused variables throws console warnings. Especially with this many (plus the ones below), it kinda pollutes the console